organizations/models.go

```diff
@@ -172,3 +174,18 @@ type ChatSettings struct {
 	MediaVisibility bool   `json:"media_visibility" bson:"media_visibility"`
 	FontSize        string `json:"font_size" bson:"font_size"`
 }
+
+
+// gets the details of a member in a workspace using parameters such as email, username etc
+// returns parameters based on the member struct
+func FetchMember(filter map[string]interface{}) (*Member, error) {
+	member_collection := "members"
+	member := &Member{}
+	memberCollection, err := utils.GetMongoDbCollection(os.Getenv("DB_NAME"), member_collection)
```

Comment:
You didn't take into consideration the organization. Why? You want to Grego from HNG7 workspace to be owner in HNG8 without being in HNG8?

Reply:
Yes........

I just knocked myself right now....like stupid stupid stupid